src/main/java/at/hannibal2/skyhanni/config/features/event/hoppity/HoppityWaypointsConfig.kt

```diff
@@ -74,10 +74,17 @@ class HoppityWaypointsConfig {
     var showNearbyDuplicates: Boolean = false
 
     @Expose
+    //#if FORGE
     @ConfigOption(
         name = "Load from NEU PV",
         desc = "Load Hoppity Egg Location data from API when opening the NEU Profile Viewer."
     )
+    //#else
+    //$$@ConfigOption(
+    //$$    name = "Load from SkyBlock PV",
+    //$$    desc = "Load Hoppity Egg Location data from API when opening the SkyBlock Profile Viewer."
```

Comment:
```suggestion
    //$$    desc = "Load Hoppity Egg Location data from API when opening the SkyBlock Profile Viewer mod."
```
To not make it sound like an official Hypixel thing, especially since they do have a bare bones click to view profile thing.